app/Repositories/LocationRepository.php

```diff
@@ -92,9 +95,9 @@ public function getDatesOfFilter($filter)
         return $array_dates;
     }
 
-    public function analystic($full_time_seat, $shiftData, $filter, $title, $className)
+    public function analystic($fullTimeSeat, $shiftData, $filter, $title, $className, $totalSeat)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Variable "array_dates" is not in valid camel caps format